content/en/blog/_posts/2019-09-18-kubernetes-1-16-release-announcement.md

```diff
@@ -114,6 +114,19 @@ Endpoint Slices also include greater flexibility for address types. Each contain
 
 The Kubernetes documentation has a lot more information about Endpoint Slices. There’s also a [great KubeCon talk](https://www.youtube.com/watch?v=Y5JOCCbJ_Fg) that provides more information on the initial rationale for developing Endpoint Slices. As an alpha feature in Kubernetes 1.16, they will not be enabled by default, but the docs cover how to enable them in your cluster.
 
+## Significant Changes to the `Kubernetes API`
+
+As the Kubernetes API has evolved many of the api objects that we rely on have been promoted to stable or been reorganized to different groups. We mark older APIs deprecated and make newer versions available. 
+
+An example of this is the `Deployment` object. This was initially created under the `extensions/v1beta1` group in 1.6 and as the project changed has been promoted to `extensions/v1beta2`, `apps/v1beta2` and finally `apps/v1` in 1.9. 
```

Comment:
Maybe link to Deployment, ie https://kubernetes.io/docs/concepts/workloads/controllers/deployment/
What do you think @mauilion?

Reply:
addressed

Reply:
Thanks for the review!